### DISCUSSION TITLE
Atlas Scientific errorn 22

### DISCUSSION THREAD

**Discussion by Cinett87** [theyosh/TerrariumPI#749] [Q&A] [answered]
getting error sensor [errorn 22] "temp1" at address "66" invalid argument I have also attempted to add it as  0x66, 66,1 and 0x66,1 all have returned invalid argument I have rebooted and restarted many times. 
![hardware used](https://user-images.githubusercontent.com/115588020/203392486-dc3d67b1-ec40-4512-9a0b-82d2436f430b.jpg)
![i2cdetect](https://user-images.githubusercontent.com/115588020/203392488-7683b709-0a13-4bf1-bb41-024b9af51145.jpg)

**Comment by theyosh** [comment, marked as answer]
Hi, could we discus here? https://github.com/theyosh/TerrariumPI/issues/753

**Reply by theyosh** [reply]
I need testing from you side. Can you provide some input at #753
